### DISCUSSION TITLE
LaTeX template customisations not working

### DISCUSSION THREAD

**Discussion by LandLordiusTH31** [xournalpp/xournalpp#5526] [Q&A] [answered]
Hi all
After editing the default_template.tex file as described in the [guide](https://xournalpp.github.io/guide/tools/latex/#template), I've tried executing the same example formulae but it gives me the "formula is empty when rendered or invalid" text, suggesting an error has occured.

I am very certain the basic installation of LaTeX is correct as running basic math formulas works just fine. I only have trouble executing package specific stuff.

I have attached my .tex file and a relevant screenshot for additional context.
Thanks in advance.
[edited_default_template.txt](https://github.com/xournalpp/xournalpp/files/14424910/edited_default_template.txt)
![image](https://github.com/xournalpp/xournalpp/assets/103369672/635caf1e-6edc-4705-b209-42368a7ac7bf)

**Comment by tattsan** [comment, marked as answer]
You've added your settings after `\end{document}`, but you should put them before `\begin{document}`.

**Reply by LandLordiusTH31** [reply]
Much thanks, it's all been fixed
